src/index.tsx

```diff
@@ -4,8 +4,7 @@ import { NexusClient, createNexusClient } from "@bbp/nexus-sdk";
 import { NexusProvider } from "@bbp/react-nexus";
 import { setUpSession, setToken } from "./utils/auth";
 import Header from "./components/Header";
-import Filters from "./containers/Filters";
-import Results from "./components/Results";
+import Workspace from "./containers/Workspace";
```

Comment:
can you move this into a `./SparqlView.tsx` file instead? Worspace doesn't mean much to me